static/js/lib/ckeditor/plugins/SubscriptSyntaxPlugin.tsx

```diff
@@ -0,0 +1,47 @@
+import Turndown from "turndown"
+import Showdown from "showdown"
+
+import MarkdownSyntaxPlugin from "./MarkdownSyntaxPlugin"
+import { TurndownRule } from "../../../types/ckeditor_markdown"
+
+
+export default class SubscriptSyntaxPlugin extends MarkdownSyntaxPlugin {
+  static get pluginName(): string {
+    return "SubscriptSyntaxPlugin"
+  }
+
+
+  get showdownExtension() {
+    return function subscriptExtension(): Showdown.ShowdownExtension[] {
+      return [
+        {
+          type:    "lang",
+          // eslint-disable-next-line no-useless-escape
+          regex:   new RegExp(/\\\(\s+_{\w+}_\s+\\\)/),
+          replace: (s :string) => {
+            const value = s.match("(?<=_{)(.*)(?=}_)")
+            // eslint-disable-next-line no-debugger
+            return `<sub class='subscript'>${value ? value[0] : ""}</sub>`
+          }
+        }
+      ]
+    }
+  }
+
+  get turndownRules(): TurndownRule[] {
+    return [
+      {
+        name: "SubscriptSyntaxPlugin",
+        rule: {
+          filter: function(node) {
+            return (node.nodeName === "SUB")
+          },
+          replacement: (content: string, _: Turndown.Node): string => {
+            const v = `\\( _{${content}}_ \\)`
```

Comment:
I don't think there should be a trailing underscore. Just `_{$content}}`. (This version works fine in studio, but to get the shortcodes to work in Hugo I had to remove the trailing underscore.)

Reply:
done